### PR TITLE
Revert "Add projections for .cljc files"

### PR DESCRIPTION
**Pull request by gonzaloserrano**
Reverts tpope/vim-salve#14

Otherwise going back from the test does not work.

### CODE REVIEW

**Comment by jivagoalves** [comment]
I'm experiencing the same issue.

File: plugin/salve.vim [hunk 1]
```diff
@@ -220,40 +219,27 @@ function! s:projectionist_detect() abort
   let projections = {}
 
   let proj = {'type': 'test', 'alternate': map(copy(main), 'v:val."/{}.clj"')}
-  let proj = {'type': 'test', 'alternate': map(copy(main), 'v:val."/{}.cljc"')}
```
Comment (tpope) [inline plugin/salve.vim]: This is the real problem, I believe. Changing it to something like `map(copy(main), 'v:val."/{}.cljc"') + map(copy(main), 'v:val."/{}.clj"')` should fix it without throwing out the baby with the bathwater.

**Comment by tpope** [comment]
Added a commit comment with what I think should be a more precise fix.

**Comment by tpope** [comment]
Fixed by #17.

**Pull request closed without merging**